docs/administration/configuration/system-properties.md

```diff
@@ -37,8 +35,8 @@ java -server -Dserver.servlet.session.timeout=3600 -Dserver.port=8080 -jar runde
 - `rundeck.jetty.connector.ssl.includedProtocols` Comma-separated list of SSL protocols to include. Default is based on available protocols. See [Disabling SSL Protocols](/administration/security/ssl.md#disabling-ssl-protocols).
 - `rundeck.jetty.connector.ssl.excludedCipherSuites` Comma-separated list of Cipher suites to disable. No default. See [Disabling SSL Protocols](/administration/security/ssl.md#disabling-ssl-protocols).
 - `rundeck.jetty.connector.ssl.includedCipherSuites` Comma-separated list of Cipher suites to enable. Default is based on available cipher suites. See [Disabling SSL Protocols](/administration/security/ssl.md#disabling-ssl-protocols).
-- `logging.config` The absolute path to your log4j2 configuraiton file. This is the Spring Boot log file setting and must be set in rpm,deb, and docker environments for the logging system to initialize properly (Rundeck 3.3.x)
-- `log4j.configurationFile` The absolute path to your log4j2 configuration file. (Rundeck 3.3.x)
+- `logging.config` The absolute path to your log4j2 configuration file. This is the Spring Boot log file setting and must be set in rpm,deb, and docker environments for the logging system to initialize properly
```

Comment:
@sjrd218 can you clarify what this section says? are both config values necessary to set the logfile path? `logging.conf` vs `log4j.configurationFile` ?

Reply:
We should be good to remove reference to the log4j.configurationFile. The Spring boot config is the one we would want to document.